service/controller.go

```diff
@@ -1699,24 +1700,31 @@ func (s *service) ControllerGetVolume(ctx context.Context,
 
 	//check if volume exists
 	if !isiConfig.isiSvc.IsVolumeExistent(ctx, isiPath, volName, "") {
+		abnormal = true
+		message = fmt.Sprintf("volume does not exists at this path %v", isiPath)
```

Comment:
why are we not returning from here when the volume is abnormal. Does it make sense to proceed with other checks ?

Reply:
we are not checking anything else, the second if condition will be executed only if volume is not abnormal
It's just we have a common return statement for both these abnormal cases

Reply:
change exists to exist. (volume does not exist at this path %v)